de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/ProjectNegotiationObservable.java

```diff
@@ -116,4 +116,15 @@ public synchronized void remove(ProjectNegotiation negotiation) {
 
         return currentNegotiations;
     }
+
+    /**
+     * @return true if running negotiations exist
```

Comment:
Yeah I know the method description will be the same as the return description. However, just add it.

Reply:
The official oracle doc guide states this from the reversed situation, but the outcome is the same:
> include it for all other methods, even if its content is entirely redundant with the method description

https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@return